### DISCUSSION TITLE
[Lab5] run_vivado_gcd.sh 沒有產生.bit file

### DISCUSSION THREAD

**Discussion by Fufaat** [bol-edu/HLS-SOC-Discussions#151] [Q&A]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/101532335/22054db0-1506-4195-841f-39038fb6bd2a)
如上圖，我在執行完檔案之後有產生.hwh file但沒有.bit file，請問有可能是什麼原因嗎?謝謝大家。

**Comment by LingChiYang** [comment]
同學好

大概率是記憶體不夠，建議清空記憶體在執行
